runtime/rococo/src/lib.rs

```diff
@@ -645,7 +645,8 @@ parameter_types! {
 /// Type to convert an `Origin` type value into a `MultiLocation` value which represents an interior location
 /// of this chain.
 pub type LocalOriginToLocation = (
-	// We allow an origin from the Collective pallet to be used in XCM as a corresponding Plurality
+	// We allow an origin from the Collective pallet to be used in XCM as a corresponding Plurality of the
+	// `Unit` body.
```

Comment:
Means we can close this?

https://github.com/paritytech/substrate/pull/8525

Reply:
Hell no. #8525 is still going to be super useful to allow e.g. the council to own tokens on its own chain (or on another chain, in the case that the other chain doesn't recognise its `Plurality` origins).

This PR is useful for the case where the other chain does recognise the `Plurality` origin and has stuff like `EnsureXcm` to use it.